### PR TITLE
Added create order functionality

### PR DESCRIPTION
**Pull request by antonmaskalik**

### CODE REVIEW

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 1]
```diff
@@ -75,7 +76,40 @@ public List<Product> GetTop5()
...
+
+            }).Start();
+
+            new Thread(() =>
+            {
+                Thread.Sleep(120000);
+
+                store.PurchasedGoods.Clear();
+                store.PrintAllProducts(store.PurchasedGoods);
+
```
[the remark this answers is not in the document]
Comment (antonmaskalik) [reply OnlineStore/StoreHelpers/StoreHelper.cs:109]: Done

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 2]
```diff
@@ -99,17 +100,18 @@ public void CreateOrder(string nameProduct)
...
             }).Start();
 
-            new Thread(() =>
-            {
-                Thread.Sleep(120000);
-
-                store.PurchasedGoods.Clear();
-                store.PrintAllProducts(store.PurchasedGoods);
+            TimerCallback tm = new TimerCallback(CleanPurchasedGoods);
+            Timer timer = new Timer(tm, null, 120000, 120000);
```
Comment (MarinaGotka) [inline OnlineStore/StoreHelpers/StoreHelper.cs:107]: This should be in separate thread

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 3]
```diff
@@ -81,37 +83,47 @@ public List<Product> GetTop5()
...
         public void CreateOrder(string nameProduct)
         {
-            new Thread(() =>
+            ThreadPool.SetMaxThreads(10, 0);
+
+            new Thread(new ThreadStart(() =>
             {
-                Console.WriteLine("Started thread {0}", Thread.GetDomainID().ToString());
+                Console.WriteLine("Started thread {0}", Thread.CurrentThread.ManagedThreadId);
+                Console.WriteLine("Thread coount {0}", Process.GetCurrentProcess().Threads.Count);
```
Comment (antonmaskalik) [inline OnlineStore/StoreHelpers/StoreHelper.cs:91]: remove it please

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 4]
```diff
@@ -81,37 +83,47 @@ public List<Product> GetTop5()
...
                 Random random = new Random();
-                int time = random.Next(1, 30);
-
-                List<Product> allProducts = new List<Product>(store.GetAllProducts());
+               int time = random.Next(30, 100);
```
Comment (antonmaskalik) [inline OnlineStore/StoreHelpers/StoreHelper.cs:93]: should be from 1 to 30

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 5]
```diff
@@ -81,37 +83,47 @@ public List<Product> GetTop5()
...
 
-                Product SelectedProduct = allProducts.Find(p => p.Name == nameProduct);
+               List<Product> allProducts = new List<Product>(store.GetAllProducts());
```
Comment (antonmaskalik) [inline OnlineStore/StoreHelpers/StoreHelper.cs:95]: this is unnecessary here

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 6]
```diff
@@ -81,37 +83,47 @@ public List<Product> GetTop5()
...
+               {
+                   store.PurchasedGoods = new List<Product>();
+               }
 
-            TimerCallback tm = new TimerCallback(CleanPurchasedGoods);
-            Timer timer = new Timer(tm, null, 120000, 120000);
+               store.PurchasedGoods.Add(SelectedProduct);
+               store.PrintAllProducts(store.PurchasedGoods);
+                Console.WriteLine("Finished thread {0}", Thread.CurrentThread.ManagedThreadId);
+                Console.WriteLine("Thread coount {0}", Process.GetCurrentProcess().Threads.Count);
```
Comment (antonmaskalik) [inline OnlineStore/StoreHelpers/StoreHelper.cs:109]: remove it please

File: OnlineStore/StoreHelpers/StoreHelper.cs [hunk 7]
```diff
@@ -81,37 +83,47 @@ public List<Product> GetTop5()
...
+            })).Start();
         }
 
         public void CleanPurchasedGoods(object obj)
         {
             store.PurchasedGoods.Clear();
-            store.PrintAllProducts(store.PurchasedGoods);
             Console.WriteLine("Cleaned list PurchasedGoods!");
+            Console.WriteLine("Thread coount {0}", Process.GetCurrentProcess().Threads.Count);
```
Comment (antonmaskalik) [inline OnlineStore/StoreHelpers/StoreHelper.cs:117]: remove it

**Pull request merged**